js/README.md

```diff
@@ -339,7 +339,7 @@ This project provides an ONNX Runtime React Native JavaScript library to run ONN
 
 ### Models with ORT format
 
-By default, ONNX Runtime React Native leverages ONNX Runtime Mobile package with ORT format. Follow the [instruciton](https://onnxruntime.ai/docs/tutorials/mobile/model-conversion.html) to covert ONNX model to ORT format.
+By default, ONNX Runtime React Native leverages ONNX Runtime Mobile package with ORT format. Follow the [instructions](https://onnxruntime.ai/docs/reference/ort-format-models.html) to convert ONNX model to ORT format.
```

Comment:
Changed the URL for this comment based on context. Originally pointed to tutorial for mobile, but seems like it wanted to reference .ort file description at https://onnxruntime.ai/docs/reference/ort-format-models.html